plugins/SegmentEditor/vue/src/SegmentGenerator/SegmentGenerator.vue

```diff
@@ -362,6 +362,10 @@ export default defineComponent({
           format: 'json',
           method: 'API.getSuggestedValuesForSegment',
           segmentName: orCondition.segment,
+          idSite: this.idsite,
```

Comment:
This was actually a bug, that caused the suggested segment values to always be requested for the current idSite instead of the one provided for the component.